src/components/input/demoBasicUsage/index.html

```diff
@@ -57,7 +58,7 @@
         </md-input-container>
       </div>
 
-      <md-input-container flex>
+      <md-input-container layout="column">
```

Comment:
Ah, I meant to swap this out.  This is just a hack to say that this is a block item.  I changed other examples to a class called `block`